page/specification.php

```diff
@@ -0,0 +1,89 @@
+<?php
+    $id_produk = $_GET['id_produk'];
+    $select = mysqli_query($connection, "select * from produk where id_produk=$id_produk");
+    while($data = mysqli_fetch_array($select)){
+        $nama_produk = $data["nama_produk"];
+        $jenis_produk = $data["jenis_produk"];
+        $kategori = $data["kategori"];
+        $gambar = $data["gambar"];
+        $video = $data["video"];
+        $spek = $data["spek"];
+        $deskripsi = $data["deskripsi"];
+    }
+?>
+<link rel="stylesheet" href="css/template-2.css">
+
+<div class="full text-center">
+    <span class="garis"></span>
+    <h1 class="display-5 text-light"><dt><b><?php echo $nama_produk; ?></b></dt></h1>
+</div>
+<nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item font-weight-bold"><a href="index.php">Pindad</a></li>
+        <li class="breadcrumb-item font-weight-bold"><a href="">Produk & Jasa</a></li>
+        <li class="breadcrumb-item font-weight-bold"><a href=""><?php echo $jenis_produk ?></a></li>
+        <li class="breadcrumb-item active" aria-current="page"><?php echo $nama_produk ?></li>
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)